cpp/dolfinx/fem/utils.h

```diff
@@ -86,6 +91,34 @@ get_cell_facet_pairs(std::int32_t f, std::span<const std::int32_t> cells,
 
   return cell_local_facet_pairs;
 }
+
+/// Helper function to get an array of of (cell, local_vertex) pairs
+/// corresponding to a given vertex index.
```

Comment:
Add a note in the documentation that if a vertex is connected to multiple cells (which is almost always the case),
this function picks the first one in `cells`.

Just good to have in main docs for later extensions:)